specs/stork-deployment.yaml

```
@@ -8,22 +8,23 @@ data:
   # of Kubernetes, so please update those as required. The requirement to manually specify this
   # list will go away with Kubernetes v1.10 where it will use the defaults if nothing is
   # specified.
-  policy.cfg: |-
-    {
-      "kind": "Policy",
-      "apiVersion": "v1",
-      "extenders": [
-        {
-          "urlPrefix": "http://stork-service.kube-system.svc.cluster.local:8099",
-          "apiVersion": "v1beta1",
-          "filterVerb": "filter",
-          "prioritizeVerb": "prioritize",
-          "weight": 5,
-          "enableHttps": false,
-          "nodeCacheCapable": false
-        }
-      ]
-    }
+  stork-config.yaml: |-
+    apiVersion: kubescheduler.config.k8s.io/v1beta2
+    kind: KubeSchedulerConfiguration
+    leaderElection:
```

Don't you need the other "Required" fields like ResourceLock from here: https://kubernetes.io/docs/reference/config-api/kube-scheduler-config.v1beta3/#LeaderElectionConfiguration

---

I think it is, I just found validation code for this:

https://github.com/kubernetes/kubernetes/commit/0334a34e4af9b56ffa4d8fe17514c931c69db84b#diff-9f70f827e156498d9206aca249a27a6e508861dc95121540e17d4f806de26514R42-R44

I just took default values from test:
https://github.com/kubernetes/kubernetes/blob/a266805f33197ba17577e2e71511f68ce9db57eb/cmd/kube-scheduler/app/options/options_test.go#L382-L390

```suggestion
    leaderElection:
        leaseDuration: 15s
        renewDeadline: 10s
        retryPeriod: 2s
        resourceLock: leases
```